packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -65,9 +69,13 @@ export function makeVatKeeper(
   vatID,
   addKernelObject,
   addKernelPromiseForVat,
+  kernelObjectExists,
   incrementRefCount,
   decrementRefCount,
+  getObjectRefCount,
+  setObjectRefCount,
   getReachableAndVatSlot,
+  addMaybeFreeKref,
```

Comment:
For some reason I'm thinking of the famous Alan Perlis quote: "If you have a procedure with 10 parameters, you probably missed some".  I'm wondering if some refactoring is called for here; 16 parameters seems awfully unwieldy.

Reply:
oh yeah. after this PR though :)